AzureCalling/Controllers/AudioDeviceSelectionViewController.swift

```diff
@@ -0,0 +1,96 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import UIKit
+
+class AudioDeviceSelectionViewController: UIViewController, UITableViewDelegate {
+
+    // MARK: Properties
+
+    var audioDeviceSelectionManager: AudioDeviceSelectionManager!
+    var audioDeviceTableDataSource: TableViewDataSource!
+    var deviceDrawerConstraint: NSLayoutConstraint?
+
+    // MARK: IBOutlets
+
+    @IBOutlet weak var deviceDrawer: UITableView!
+
+    // MARK: UIViewController events
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        view.backgroundColor = UIColor(red: 0, green: 0, blue: 0, alpha: 0.8)
+        view.isOpaque = false
+
+        let audioDeviceCell = UINib(nibName: "BottomDrawerCellView",
+                                      bundle: nil)
+        deviceDrawer.layer.cornerRadius = 8
+        deviceDrawer.register(audioDeviceCell, forCellReuseIdentifier: "BottomDrawerCellView")
+        deviceDrawer.dataSource = audioDeviceTableDataSource
+        deviceDrawer.delegate = self
+        audioDeviceTableDataSource?.deselectAllRows()
+        let currentAudioDevice = audioDeviceSelectionManager.getCurrentAudioDevice()
+        audioDeviceTableDataSource?.selectRow(title: currentAudioDevice.name)
+        deviceDrawer.reloadData()
+        hideDeviceDrawer()
+        showDeviceDrawer()
+    }
+
+    // MARK: Private Functions
+
+    private func hideDeviceDrawer() {
```

Comment:
For hide action, could we just do view remove rather than constraint change?
``
self.view.removeFromSuperView()

``

Reply:
We need to implement the initial position such that it will slide up from the bottom.  So perhaps it was mis-named as I wanted to put it below the current view rather than hide it